dev/src/react-dat-gui/components/DatFolder.js

```diff
@@ -22,17 +23,9 @@ export default class DatFolder extends Component {
     this.setState({ closed });
   }
 
-  renderChildren() {
-    // Disable this rule to take title out of the props so nested folders can have unique titles.
-    // eslint-disable-next-line no-unused-vars
-    const { children, title, ...rest } = this.props;
-
-    return React.Children.map(children, child => cloneElement(child, {...rest}));
```

Comment:
Have you tested unique titles in nested folders after removing this?

Reply:
Yes, `title` is a required prop for `DatFolder` component, so you can pass which title you want in the props:
```
<DatFolder title="Folder">
  <DatFolder title="Nested Folder">
    <DatFolder title=""Another Nested Folder""></DatFolder>
  </DatFolder>
</DatFolder>
```

Reply:
Yah, I wrote the original component, could have sworn that that wasn’t working for me by simply rendering children, I had to explicitly remove the title prop with spread. I’ll check out your PR and try again.

Reply:
@rohan-deshpande any updates?

Reply:
Sorry! I'll get to it this week for sure

Reply:
Hey sorry for the delay in checking this out. It seems to all be fine!

It might be nice to add this feature to one of the nested folders in the example to demo it. For example I've just done this 

```jsx 
<DatFolder title="Folder">
            <DatString path="string" label="String" />
            <DatNumber
              path="minMaxNumber"
              label="Number"
              min={0}
              max={100}
              step={1}
            />
            <DatFolder title="Opened Nested Folder" closed={false}>
              <DatNumber
                path="minMaxNumber"
                label="Number"
                min={0}
                max={100}
                step={1}
              />
              <DatString path="nested.string" label="String" />
              <DatFolder title="Another Nested Folder">
                <DatColor label="Color" path="color" />
                <DatString path="nested.string" label="Nested String" />
              </DatFolder>
            </DatFolder>
          </DatFolder>
```

And the result is ![screen shot 2018-12-07 at 12 06 49 pm](https://user-images.githubusercontent.com/1079637/49621302-bfdeaa00-fa18-11e8-8ae7-335bcb479171.png)

@claus do you mind adding me to contributors so I can merge PRs?

Reply:
@claus I believe this is ready to be merged in.